src/Symfony/Component/Security/Core/User/User.php

```diff
@@ -118,6 +120,11 @@ public function eraseCredentials()
     {
     }
 
+    public function getMetadata()
```

Comment:
ununsed, isn't it? is it really needed?

Reply:
This is how you get the information from the user, if we don't have this getter we can't get the populated metadata